pyproject.toml

```diff
@@ -0,0 +1,71 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "python-multipart"
+version = "0.0.5"
+description = "A streaming multipart parser for Python"
+authors = [
+    {name = "Andrew Dunham"},
+]
+readme = "README.rst"
+requires-python = ">=3.7"
+license = {text = "Apache"}
+classifiers = [
+        'Development Status :: 5 - Production/Stable',
+        'Environment :: Web Environment',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: Apache Software License',
+        'Operating System :: OS Independent',
+        'Programming Language :: Python :: 3 :: Only',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
+        'Programming Language :: Python :: 3.9',
+        'Programming Language :: Python :: 3.10',
+        'Programming Language :: Python :: 3.11',
+        'Topic :: Software Development :: Libraries :: Python Modules'
+]
+dependencies = [
+    "atomicwrites==1.2.1",
+    "attrs==19.2.0",
+    "coverage==4.5.1",
+    "more-itertools==4.3.0",
+    "pbr==4.3.0",
+    "pluggy==1.0.0",
+    "py==1.11.0",
+    "pytest==7.2.0",
+    "PyYAML==5.1",
+]
+
+[project.urls]
+"Homepage" = 'https://github.com/andrew-d/python-multipart'
+
+[project.optional-dependencies]
+dev = [
```

Comment:
If we were to pin, here would be better as it will only be done if we're setting up a dev enviornment, not a library usage environment